src/Internal/Plutus/Types/Credential.purs

```diff
@@ -68,23 +73,18 @@ instance
 
 -- NOTE: mlabs-haskell/purescript-bridge generated and applied here
```

Comment:
Does this note still apply?

Reply:
Nope. All notes like this were stripped. Thank you.